### ISSUE TITLE
cmake error in cmake version of 3.10.2 (==> 3.15.5 is fine)

### ISSUE THREAD

**Issue by imaihal** [clang-ykt/ONNF#20]
With cmake version 3.10.2, cmake of ONNF failed after applying #12 with the following error message: 
```
  CMake Error at doc/doc_check/CMakeLists.txt:1 (find_package):
  By not providing "FindPython3.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Python3", but
  CMake did not find one.

  Could not find a package configuration file provided by "Python3" with any
  of the following names:

    Python3Config.cmake
    python3-config.cmake

  Add the installation prefix of "Python3" to CMAKE_PREFIX_PATH or set
  "Python3_DIR" to a directory containing one of the above files.  If
  "Python3" provides a separate development package or SDK, be sure it has
  been installed.
```

I confirmed cmake version 3.15.5 is fine. If there is a prerequisite version, I think it is better to write it somewhere.

**Comment by tungld** [comment]
I got the same problem with Cmake version 3.10.2. I guess the problem is that the current way to find Python3 does not work with virtual environments (in my case, I am using python in anaconda).

I fixed this problem by changing the following part in `doc/doc_check/CMakeLists.txt`:
```
find_package(Python3 REQUIRED
                        COMPONENTS Interpreter)
```
to
```
find_package(PythonLibs 3)
```

**Comment by tungld** [comment]
@tjingrant Could you please look at this issues?

**Comment by imaihal** [comment]
This fix works both 3.15.5 and 3.10.2 in my environment.
